doc/operator/deploy-operator.md

```diff
@@ -0,0 +1,185 @@
+# Observatorium Operator
+![Observatorium](https://avatars0.githubusercontent.com/u/51818702?s=100&v=4sanitize=true)
+
+## Prelude
+
+Check the following resources for more information about Observatorium:
+* [Documentation repository](https://github.com/observatorium/docs/)
+* [Observatorium API](https://github.com/observatorium/observatorium/)
+* [Observatorium configuration repository](https://github.com/observatorium/configuration)
+* [Locutus - The framework which the operator is based on](https://github.com/brancz/locutus)
+
+
+## How to deploy - Kubernetes and OpenShift
+
+In order to ease the installation of Observatorium, an operator is available.
+
+### Prerequisites
+
+#### Create Namespaces
+```shell script
+kubectl create namespace minio
+kubectl create namespace observatorium
+```
+
+
+#### S3 storage endpoint and secret
+For **testing purposes** you may use [minio](https://github.com/minio/minio) as describe below.
+```shell script
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/environments/dev/manifests/minio-secret.yaml
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/environments/dev/manifests/minio-pvc.yaml
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/environments/dev/manifests/minio-deployment.yaml
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/environments/dev/manifests/minio-service.yaml
+```
+
+### Deployment
+
+#### Prometheus CRDs - Kubernetes Only
```

Comment:
Maybe elaborate what you mean by "Kubernetes Only". I assume you mean that on OpenShift these already exist so it's not necessary there. I don't think that's clear enough right now.

Reply:
fair point.
It is already included in OpenShift because of the monitoring stack, hence k8s only.
I'll rephrase that.

Reply:
Done